google/resource_compute_instance_template.go

```diff
@@ -262,8 +262,8 @@ func resourceComputeInstanceTemplate() *schema.Resource {
 						"automatic_restart": &schema.Schema{
 							Type:     schema.TypeBool,
 							Optional: true,
-							Default:  true,
 							ForceNew: true,
+							Computed: true,
```

Comment:
If you set this field to Computed, it means a user cannot specify a value for `automatic_restart`. GCP default value for this field is true. It means a user could never create an instance with terraform with `automatic_restart = false`. I am confused about why that would be something desirable